packages/core/src/App/Containers/RealAccountSignup/currency-selector.jsx

```diff
@@ -21,6 +21,21 @@ import 'Sass/currency-select-radio.scss';
 
 // Radio input
 export const RadioButton = ({ field: { name, value, onChange, onBlur }, id, label, className, ...props }) => {
+    const popover_message = /^UST$/i.test(id) ? (
```

Comment:
is API sending UST or USDT?

Reply:
Both and we need to show popovers for `UST` and `USDT`. According to the implementation on line 62, if it's not `UST` then it's `USDT`.